internal/karma/README.md

```diff
@@ -0,0 +1,658 @@
+# Installing the Karma rules
+
+Add the `@bazel/karma` npm package to your `devDependencies` in `package.json`.
+
+Your `WORKSPACE` should declare a `yarn_install` or `npm_install` rule named `npm`.
+It should then install the rules found in the npm packages.
+
+```python
+yarn_install(
+  name = "npm",
+  package_json = "//:package.json",
+  yarn_lock = "//:yarn.lock",
+)
+
+# Install all Bazel dependencies needed for npm packages that supply Bazel rules
+load("@npm//:install_bazel_dependencies.bzl", "install_bazel_dependencies")
+install_bazel_dependencies()
+```
+
+This causes the `@bazel/karma` package to be installed as a Bazel workspace named `npm_bazel_karma`.
+
+Now add this to your `WORKSPACE` to install the Karma dependencies:
+
+```python
+# Fetch transitive Bazel dependencies of npm_bazel_karma
+# ONLY REQUIRED if you are using the @bazel/karma npm package
+load("@npm_bazel_karma//:package.bzl", "rules_karma_dependencies")
+rules_karma_dependencies()
+```
+
+This installs the `build_bazel_rules_nodejs` and `io_bazel_rules_webtesting` repositories, if you haven't installed them earlier.
```

Comment:
nit: This is slightly misleading since you'd have to have loaded `build_bazel_rules_nodejs ` manually if you are using the yarn_install method above. The rules_typescript README.md already documents this so maybe overkill to add it but we could duplicate it here to be explicit. Eventually each rules set README should be self-sufficient.

```
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

# Fetch rules_nodejs
# (you can check https://github.com/bazelbuild/rules_nodejs for a newer release than this)
http_archive(
    name = "build_bazel_rules_nodejs",
    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.18.5/rules_nodejs-0.18.5.tar.gz"],
    sha256 = "c8cd6a77433f7d3bb1f4ac87f15822aa102989f8e9eb1907ca0cad718573985b",
)

# Setup the NodeJS toolchain
load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "yarn_install")
node_repositories()
```

before the `yarn_install` above?